### ISSUE TITLE
How to pass switch to 7zip (eg -mtc=off)

### ISSUE THREAD

**Issue by fazl** [stefankueng/CryptSync#135]
Thanks for putting the work into such a useful program. I've started using it more seriously recently.

I've recently come across a problem after a pc died and I tried to download a cloud backup and compared it to an SD card backup. A large number of .7z files are showing as different when they actually contain identical data when uncompressed..

I found this post on stackoverflow (https://stackoverflow.com/a/27203097/7409029) explaining one likely cause, that 7zip by default will incorporate create/modify/access times unless you pass a switch -mtc=off to it on compression.

I have not found any issue here about this (searched for 7zip and for mtc).

Would be possible to add a field for custom "command line" parameters to pass to the 7zip ?

Thanks again, and Gruezi
Faz

**Comment by stefankueng** [comment]
cryptsync actually wants to include the file dates. That way the dates get properly restored when decompressed. CryptSync uses the file dates to determine what to sync.

**Comment by fazl** [comment]
Yes, the modification time is critically important for deciding what to sync and what to restore - but the _access time_ (when a file was last read, but not written) should probably not be included as it can mean two _different_ zip files will uncompress to _identical_ files.  I cant recall where I read about this - its part of the NTFS filesystem, and you can avoid the access time polluting the archive by passing that switch -mtc=off.

The impact can be that unnecessary syncs occur, and when someone has to do a data recovery operation it can be lots of needless headaches trying to figure out which is the correct zip file to keep and then finally find out they both had the same contents.
Kind regards
Fazl
